current-reviews-data.ipynb

```diff
@@ -897,17 +952,378 @@
+       "</script>"
+      ],
+      "text/plain": [
+       "alt.VConcatChart(...)"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "charts = []\n",
+    "for editor in editors:\n",
+    "    if editor in ignore_editors:\n",
```

Comment:
can you please in a higher up cell filter out 
ignore_editors so it's not mixed into this plot cell? that will just make things a bit cleaner.

that could look like this:

```python
# Remove TBD and editors that are no longer active
current_editors = [editor for editor in editors if editor not in ignore_editors]

```

Reply:
then you can just loop through current editors
```
for editor in current_editors:
```